elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -109,3 +109,27 @@
   height: #{$size};
   @content;
 }
+
+// This will print the $LOCAL-VARIABLES map for debugging within components
+@mixin pfe-local-debug($map: $LOCAL-VARIABLES) {
+  :root {
+      @each $property, $value in $map {
```

Comment:
I think using `@include pfe-print-local($map);` here will do the same thing and ensure the logic stays in sync

Reply:
nice -- i'll update that!